extensions/wikia/WikiaHubsV3/hooks/WikiaHubsV3Hooks.php

```diff
@@ -7,11 +7,11 @@ class WikiaHubsV3Hooks {
 	 *
 	 * @return true because it's a hook
 	 */
-	static public function onArticleFromTitle(&$title, &$article) {
+	static public function onArticleFromTitle(Title $title, &$article) {
```

Comment:
If the `$article` is an object you could remove the reference and hint a class for it. :)